data_steward/analytics/cdr_ops/retraction_validation.py

```diff
@@ -37,11 +37,25 @@
 SELECT table_name
 FROM `{{project}}.{{new_dataset}}.INFORMATION_SCHEMA.COLUMNS`
 WHERE column_name = "person_id"
-''').render(project=project_id, new_dataset=q4_dataset)
+''').render(project=project_id, new_dataset=new_dataset)
 pid_table_list = client.query(person_id_tables_query).to_dataframe().get(
     'table_name').to_list()
 pid_table_list
 
+
+def provenance_table_for(table: str, is_deidentified: str = 'false'):
+    """
+    Returns a mapping table for a domain table.
+    
+    :param table: identifies domain table name
+    :param is_deidentified: identifies if a the dataset is de-identified choose b/e true/false.
+    """
+    if is_deidentified.lower() == 'false':
+        return f'_mapping_{table}'
+    else:
+        return f'{table}_ext'
```

Comment:
Does it make sense to move this code snipped to the https://github.com/all-of-us/curation/blob/develop/data_steward/analytics/cdr_ops/notebook_utils.py module?  You've used it in two different files now.  It could be a useful helper function in other notebooks also.